gridded_etl_tools/dataset_manager.py

```diff
@@ -79,7 +79,7 @@ def __init__(
             Sets `DatasetManager.rebuild_requested`. If this parameter is set, the manager requests and parses all available data from beginning
             to end.
         custom_output_path : str, optional
-            Overrides the default path returned by `Convenience.output_path`
+            Overrides the default path returned by `Convenience.path` for Local and S3 stores.
```

Comment:
The function name is still Convenience.output_path

Reply:
We’re actually overriding both. Correcting docstring.

Reply:
If you mean the function in store it's technically StoreInterface.path not Convenience.path